api/envoy/extensions/filters/http/grpc_json_transcoder/v3/transcoder.proto

```diff
@@ -68,6 +69,9 @@ message GrpcJsonTranscoder {
     // generate JSON field names using the ``json_name`` option, or lower camel case,
     // in that order. Setting this flag will preserve the original field names. Defaults to false.
     bool preserve_proto_field_names = 4;
+
+    // If true, return all streams as newline-delimited JSON messages instead of as a comma-separated array
+    bool stream_newline_delimited = 5;
```

Comment:
Please add a release note for this new feature.

Reply:
@mattklein123 added changes [here](https://github.com/envoyproxy/envoy/pull/22868/files#diff-6f9c718224c533c13c2c0ba1d5abaab86be9d0cc73808749c77934e9f9b0d5d0R258). Let me know if there's anything else needed to get those release notes out.